src/curves/secp256k1.ts

```diff
@@ -18,3 +23,20 @@ export const getPublicKey = (
   privateKey: Uint8Array | string | bigint,
   compressed?: boolean,
 ): Buffer => Buffer.from(getSecp256k1PublicKey(privateKey, compressed));
+
+export const publicAdd = (publicKey: Buffer, tweak: Buffer): Uint8Array => {
+  const point = Point.fromHex(publicKey);
+  const newPoint = point.add(
+    // Multiplies `Point(Gx, Gy)` with the tweak
```

Comment:
Can we link a source on this?

Reply:
Simplified this a bit. Do you still want me to link something?